File: problems/3210-find-the-encrypted-string/solution.py
class Solution:
    def getEncryptedString(self, s: str, k: int) -> str:
        encrypted_string = ""
        for char in s:
            encrypted_char = chr(((ord(char) - ord('a') + k) % 26) + ord('a'))
            encrypted_string += encrypted_char
        return encrypted_string

File: problems/3210-find-the-encrypted-string/solution.java
class Solution {
    public String getEncryptedString(String s, int k) {
        StringBuilder encryptedString = new StringBuilder();
        for (char c : s.toCharArray()) {
            int newChar = ((c - 'a' + k) % 26) + 'a';
            encryptedString.append((char)newChar);
        }
        return encryptedString.toString();
    }
}